src/main/java/com/tudu/todoapp/services/UserService.java

```diff
@@ -109,4 +109,13 @@ public void deleteUserById(Long userId) {
     public Optional<User> findUserByDisplayName(String displayName) {
         return userRepository.findUserByDisplayName(displayName);
     }
+
+    public void save(User user) {
```

Comment:
This method is not needed as there is already a method, `createUser` a bit further up the class that does the same.

Please, remove the `save()` method from UserService.